worlds/witness/hints.py

```diff
@@ -168,6 +170,50 @@ def get_priority_hint_locations(world: "WitnessWorld") -> List[str]:
     return priority
 
 
+def try_getting_location_group_for_location(world: "WitnessWorld", hint_loc: Location) -> Tuple[str, str]:
+    allow_regions = world.options.vague_hints == "experimental"
+
+    possible_location_groups = {
+        k: v for k, v in world.multiworld.worlds[hint_loc.player].location_name_groups.items()
+        if hint_loc.name in v
+    }
```

Comment:
I generally prefer using more descriptive variable names, but it's probably fine here since it's in a comprehension so it kinda needs to be short

Reply:
You're right, I think I will change this